python/lib/client/dmod/client/request_clients.py

```diff
@@ -5,6 +5,7 @@
 from dmod.communication.dataset_management_message import DatasetManagementMessage, DatasetManagementResponse, \
     MaaSDatasetManagementMessage, MaaSDatasetManagementResponse, QueryType, DatasetQuery
 from dmod.communication.data_transmit_message import DataTransmitMessage, DataTransmitResponse
+from dmod.communication.maas_request.job_message import *
```

Comment:
Is there a really good reason to use a wildcard import here that im glossing over?